packages/netlify-cms-widget-list/src/ListControl.js

```diff
@@ -261,6 +261,33 @@ export default class ListControl extends React.Component {
     } else {
       this.props.validate();
     }
+    this.props.onValidateObject(this.props.forID, this.validateSize());
+  };
+
+  validateSize = () => {
```

Comment:
Since this code is very similar to https://github.com/netlify/netlify-cms/blob/2146b3605f337e808f0e54c92cd2b601296f23a6/packages/netlify-cms-widget-select/src/SelectControl.js#L58 what do you think about extracting it to a shared function?
I think we can put this under a `validations` file in:
https://github.com/netlify/netlify-cms/tree/2146b3605f337e808f0e54c92cd2b601296f23a6/packages/netlify-cms-lib-widgets/src

Reply:
the return values are a bit different, not sure how well that would work

Reply:
Let me know if you think https://github.com/netlify/netlify-cms/pull/4394/commits/fc0c5a42b53b773b8972bf51e530031ad5e0c516 makes sense.
We can revert it if not

Reply:
Going to merge for the sake of moving this forward